torch/csrc/autograd/functions/accumulate_grad.cpp

```diff
@@ -93,11 +94,8 @@ variable_list AccumulateGrad::apply_with_saved(
   saved.after(variable_copy);
   saved.after(grad_copy);
 
-  TORCH_CHECK(
-      tensor_post_acc_grad_hooks() == nullptr,
-      "compiled_autograd does not support tensor_post_acc_grad_hooks");
-
-  return variable_list();
+  // A little hack - this is only here for the purpose of hooks. It will get cleared.
+  return variable_list({fake_variable_copy});
```

Comment:
@jansel surely there's a better way to smuggle stuff out? This *does* get a fake tensor into the hook (if you set a breakpoint in `def post_acc_grad_hook` you can see it having the correct value, with grad.

Reply:
Don't smuggle it out.  Just call into a virtual method on `tensor_post_acc_grad_hooks()`.

Eager returns nothing here, compiled autograd should do the same.